WordPress/Classes/ViewRelated/Post/Revisions/RevisionsTableViewController.swift

```diff
@@ -1,4 +1,6 @@
-class RevisionsTableViewController: UITableViewController {
+class RevisionsTableViewController: UITableViewController, SelectedRevisionLoadedProtocol {
+    var selectedRevisionLoaded: SelectedRevisionBlock
```

Comment:
`selectedRevisionLoaded` sounds like a Boolean to me. Something like `onRevisionLoaded` sounds like a function/callback. Know what I mean?